pkg/apis/node/v1alpha1/types.go

```diff
@@ -92,13 +92,17 @@ type Property struct {
 
 type CustomNodeResourceStatus struct {
```

Comment:
Do we need to add some fields to indicate the policy and the scope of topology manager?

Reply:
the policy is defined as FeatureGate in Kubelet, but won't be defined in CNR for now; the reason for this requirement is for scheduler to understand, right?

Reply:
Yes, these states need to be reported to the scheduler to decide which strategy to use.

Reply:
make sense, i will add a new field here to indicate this specific info